api/everycache_api/api/schemas/cache.py

```diff
@@ -22,6 +22,13 @@ class CacheSchema(ma.SQLAlchemyAutoSchema):
     description = ma.String(validate=validate.Length(min=5), required=False)
     rating = ma.Float(dump_only=True)
 
+    @post_dump
```

Comment:
I know there is not much to do about duplicating this one, but we could at least get the common code part and put it in a function. Then call it here and in the PublicCacheSchema post_dump method.

Reply:
Yes, we should have tried reusing code in similar schemas, but we have limited time and cannot afford to refactor this

Reply:
I've actually been able to do it in #40